scrapers/FemDomEmpire.yml

```diff
@@ -0,0 +1,31 @@
+name: "FemDomEmpire"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="title"]/h3

```

Comment:
the specific regex gets also the `More Updates` as a second result
```yaml
      Title: //div[@class="videoDetails clear"]/h3
```
seems safer